File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        vector<char> a(s.begin(), s.end());
        sort(a.begin(), a.end());
        vector<char> b(t.begin(), t.end());
        sort(b.begin(), b.end());
        int c= a.size();
        int d= b.size();
        if (c!=d){
            return false;
        }
        else{
            if (a==b){
                return true;
            }
        }
        return false;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        a= list(s)
        a.sort()
        b= list(t)
        b.sort()
        if (len(a)!=len(b)):
            return False
        else:
            if (a==b):
                return True
            return False